### PR TITLE
Updating the Add template 

### PR DESCRIPTION
**Pull request by vv-monsalve**
Making GF Guide more visible and reinforcing our requirements

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_add-font.md [hunk 1]
```diff
@@ -15,18 +15,21 @@ assignees: ''
...
-- [ ] Family name is unique according to namecheck.fontdata.com
+- [ ] Family name is unique according to [namecheck.fontdata.com](https://namecheck.fontdata.com/)
 - [ ] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
 - [ ] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
 - [ ] The source files are available in the repository
-- [ ] I read, agree and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/onboarding.html)
-- [ ] I am the author of the font or the author agreed to this submission
-- [ ] I will maintain the repository
+- [ ] I am the author of the font, or the author agreed to this submission
+- [ ] I will respond for the additional job required on the font for publishing it and will maintain the repository
```
Comment (eliheuer) [inline .github/ISSUE_TEMPLATE/1_add-font.md:31]: This line could maybe be more clear, something like:

`I will maintain the repository and participate in the onboarding process`

**Review by eliheuer — APPROVED** [review]
This is an improvement for sure, thanks!

The only thing I would change it to make this line more clear:

`I will respond for the additional job required on the font for publishing it and will maintain the repository`

I suggested something like:

`I will maintain the repository and participate in the onboarding process (responding to issues, merging pull requests, etc).`

**Comment by vv-monsalve** [comment]
> I suggested something like:
> 
> `I will maintain the repository and participate in the onboarding process (responding to issues, merging pull requests, etc).`

Ty!. Done with a small extra addition.

**Comment by davelab6** [comment]
Good stuff!

**Pull request merged**